common/model-views.cpp

```diff
@@ -4205,10 +4205,36 @@ namespace rs2
         if (f.get_profile().stream_type() == RS2_STREAM_DEPTH)
             ppf.depth_stream_active = true;
 
-        auto index = f.get_profile().unique_id();
+        if (f.get_profile().stream_type() == RS2_STREAM_DEPTH || f.get_profile().stream_type() == RS2_STREAM_INFRARED)
```

Comment:
I don't like the way this is implemented at Viewer level: 
a. The UI code is extremely coupled to D400
b. There is no way to utilise this in any other application
I propose to implement this as a custom processing block, that splits Depth stream into Depth,1 and Depth,2 streams, based on an option (which to output).
I can implement it myself on top of this patch, if you agree.